prov/efa/src/rdm/rxr_ep.c

```diff
@@ -2195,6 +2195,11 @@ int rxr_endpoint(struct fid_domain *domain, struct fi_info *info,
 		rxr_ep->use_shm_for_tx = false;
 	}
 
+	rxr_ep->host_id = rxr_get_host_id(rxr_env.host_id_file);
+	if (rxr_ep->host_id) {
+		EFA_INFO(FI_LOG_EP_CTRL, "rxr_ep->host_id: i-0%lx\n", rxr_ep->host_id);
```

Comment:
format -> `i-%08lx` would be better

Reply:
I think you meant `i-0%8lx`. Addressed.

Reply:
No. 0 after `%` means fill empty with `0` instead of space.

and sorry it should be `%016lx`

Reply:
Instance id is formatted this way `i-0[16 char hex string]` - total length 19 characters. I think the correct format is `i-0%016lx`

Reply:
Addressed. `%017lx`